src/pages/monetization/app-review-process/index.mdx

```diff
@@ -132,9 +132,9 @@ A good first impression may tip the balance in your favor when the trial period
 
 ## App settings
 
-If your application has some sort of settings, the best way to present them from the UX perspective is in **Settings > Integrations & apps** in the **LiveChat Application**. This is where users intuitively look for the settings of the integrations they have installed.
+If your application has some sort of settings, the best way to present them from the UX perspective is in **Settings > Integrations & apps** in the **LiveChat or HelpDesk Applications**. This is where users intuitively look for the settings of the integrations they have installed.
 
-[Read more](/getting-started/app-guides/onboarding-and-settings/#settings) about how to configure your app settings in Developer Console.
+[Read more](/extending-agent-app#app-settings) about how to configure your app settings in Developer Console.
```

Comment:
linked changed because of the change in the Onboarding BB. For now, the rest of the description in this section seems okay, but changes are not visible yet, so maybe the `Integrations & apps` name may change